R/sql-build.R

```diff
@@ -61,15 +61,15 @@ sql_build.op_rename <- function(op, con, ...) {
 
 #' @export
 sql_build.op_arrange <- function(op, con, ...) {
-  order_vars <- translate_sql_(op$dots, con)
+  order_vars <- translate_sql_(op$dots, con, sql_context = list(arrange = TRUE))
```

Comment:
I just noticed that I named the `filter` context `where` instead of `filter`, so you're right, it makes more sense.  It's also easier to align them to the sql variable that is being loaded by `sql_translate_`, like `where_sql`